pom.xml

```diff
@@ -7,7 +7,7 @@
         <relativePath />
     </parent>
     <artifactId>junit</artifactId>
-    <version>1.24-SNAPSHOT</version>
+    <version>1.25</version>
```

Comment:
it needs to be reverted, the version should be SNAPSHOT in order to let the Maven Release plugin to operate correctly

Reply:
Fixed.